packages/browser-utils/src/metrics/utils.ts

```diff
@@ -44,6 +44,85 @@ export function startAndEndSpan(
   });
 }
 
+interface StandaloneWebVitalSpanOptions {
```

Comment:
I extracted this from INP and only left vital-specific logic in CLS and INP. Most attributes and span properties are the same for INP and CLS (+soon LCP), so it makes sense to dedupe and save a couple of bytes.